src/aspire/utils/matlab_compat.py

```diff
@@ -84,7 +84,7 @@ def __enter__(self):
             random_states.append(np.random.get_state())
 
             seed = self.seed
-            # 5489 is the default seed used by MATLAB for seed 0 !
+            # 5489 is the default seed used by M   ATLAB for seed 0 !
```

Comment:
Always go through the diff carefully before committing to make sure this doesn't happen.

Reply:
Thanks. Fixed.